src/renderer/theme.store.ts

```diff
@@ -32,42 +34,46 @@ export enum ThemeType {
 }
 
 export interface Theme {
-  id: ThemeId; // filename without .json-extension
   type: ThemeType;
-  name?: string;
-  colors?: Record<string, string>;
-  description?: string;
-  author?: string;
+  name: string;
+  colors: Record<string, string>;
+  description: string;
+  author: string;
```

Comment:
We can now make these non-optional because we are no longer post-loading the files.